deploy/fly/prod.toml

```diff
@@ -40,3 +39,15 @@ processes = []
     interval = "15s"
     restart_limit = 6
     timeout = "2s"
+  
+  [[services.http_checks]]
+    interval = 10000
+    grace_period = "5s"
+    method = "get"
+    path = "/"
+    protocol = "http"
+    restart_limit = 0
```

Comment:
@chasers wouldn't it be better to set this to a positive number so `n` consecutive failing health checks will trigger a restart?

Reply:
Eventually but I don't want to do that now so we change change any behavior of the cluster after a node is up and running.